settings.py

```diff
@@ -1261,7 +1261,7 @@ def _pkce_required(client_id):
 
 _location = lambda x: os.path.join(FILEPATH, x)
 
-IS_SAAS_ENVIRONMENT = SERVER_ENVIRONMENT in ('production', 'staging')
+IS_SAAS_ENVIRONMENT = SERVER_ENVIRONMENT in ('eu', 'production', 'staging')
```

Comment:
This feels like a good time to ask why we don't include india in this list? And is it time we do?

Reply:
I also had the same question 😅 
I think we should do it since it's also managed by SaaS and is a production like env.
Probably a question for @dannyroberts?

Reply:
Yeah I'm mainly trying to think back on if there were times where it was a good thing India was not included in this variable, and if we would similarly want EU to be excluded in those scenarios. Though conceptually, it makes sense to have all of our envs in this list so I'm fine with the change.

Reply:
We can add india in a followup PR if desired